index.js

```diff
@@ -20,8 +23,49 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+   
+    Recipe.create({title:"Pizza", level: "Amateur Chef", cuisine: "Italian", ingredients: ["flour", 'cheese', 'tomato sauce', 'pure magic']})
+    .then((recipe) => {
+      console.log(recipe.title)
+    })
+    .catch()
```

Comment:
a catch by itself does nothing, try to add something like a console.log for development to see when errors might occur.